src/script/components/Conversation/index.tsx

```diff
@@ -0,0 +1,399 @@
+/* eslint-disable */
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {
```

Comment:
Let's create a `Conversation/Conversation.tsx` and an `index.ts` that exports anything that is in `Conversation.tsx`, so that:
- we can import doing `Components/Conversation`
- and the IDE doesn't show just `index.ts` in the tab. 


I know I have done this pattern (of putting the entire code in `index.ts`) in the past, but I actually do not like it